bitnami/mysql/values.yaml

```diff
@@ -102,7 +102,7 @@ auth:
   ## @param auth.database Name for a custom database to create
   ## ref: https://github.com/bitnami/bitnami-docker-mysql/blob/master/README.md#creating-a-database-on-first-run
   ##
-  database: my_database
+  database: ""
```

Comment:
```suggestion
  database: my_database
  ## @param auth.createDatabase Wheter to create the database or not
  ## ref: https://github.com/bitnami/bitnami-docker-mysql/blob/master/README.md#creating-a-database-on-first-run
  ##
  database: true
```

Reply:
Yeah it can handle backward compatibility, but needs more changes
like add schema validation to check if `.createDatabase` is `true` then database shouldn't be empty or something else like:

```
            {{- if .Values.auth.createDatabase }}
            - name: MYSQL_DATABASE
              value: {{ .Values.auth.database | quote }}
            {{- end }}
            ```

Reply:
okay but in template file it needs to check if both `.Values.auth.database` and `.Values.auth.createDatabase` is set. something like:
```
            {{- if and .Values.auth.createDatabase .Values.auth.database }}
            - name: MYSQL_DATABASE
              value: {{ .Values.auth.database | quote }}
            {{- end }}
```

I can do this and values file becomes something like:
```yaml
auth:
  ## @param auth.createDatabase Wheter to create the .Values.auth.database or not
  ## ref: https://github.com/bitnami/bitnami-docker-mysql/blob/master/README.md#creating-a-database-on-first-run
  ##
  createDatabase: false
  ## @param auth.database Name for a custom database to create
  ## ref: https://github.com/bitnami/bitnami-docker-mysql/blob/master/README.md#creating-a-database-on-first-run
  ##
  database: ""
```
also, should I add a conditional check in schema validation file to check for something like if `.Values.auth.createDatabase` is set then `.Values.auth.database` is not empty.

Reply:
Yes, there are more changes to be implemented in order to support that, the key point is that the default behavior of the Helm chart shouldn't be modified so the database with that name should be created when installing the chart from scratch using the default parameters.